src/app/workflow/info-tab/info-tab.component.ts

```diff
@@ -132,6 +133,12 @@ export class InfoTabComponent extends EntryTab implements OnInit, OnChanges {
       this.workflowsService.getWorkflowVersionOrcidAuthors(this.workflow.id, this.selectedVersion.id).subscribe((orcidAuthors) => {
         this.authors = [...this.selectedVersion.authors, ...orcidAuthors];
       });
+      if (this.selectedVersion.descriptionSource === 'CUSTOM_README') {
```

Comment:
I think it would be better to move this logic into the `BaseUrlPipe` so that if the pipe is used else where in the future, we're aware that the custom readme path can affect the base url. Maybe add a `customReadMePath` parameter to the pipe - this way, we can also create some tests for the custom readme path in https://github.com/dockstore/dockstore-ui2/blob/develop/src/app/shared/entry/base-url.pipe.spec.ts

Reply:
A wrinkle is that notebooks can also contain markdown, which can reference images using relative URLs.  The base URL for a notebook's markdown is probably similar to that of a custom `README` (the repo info + the directory portion of the notebook path), so if we can account for that in our pipe mods, that would be great.

Reply:
To account for notebook markdown files, I generalized the parameter name in `BaseUrlPipe` to `customPath` instead of `readMePath`